### ISSUE TITLE
M03-Configure Pipelines as Code with YAML -- Problem back again -- Can't do lab

### ISSUE THREAD

**Issue by ElaMark** [MicrosoftLearning/AZ400-DesigningandImplementingMicrosoftDevOpsSolutions#576]
### Contact Details

m.walsh.1@outlook.com

### What happened?

Please refer to issue #563. 

No longer can the previous issue (#563) be circumvented by using an F1 App Service Plan. Azure Passes are not allowing students to create any type of App Service Plan.

I realise that this is not a bug in the labs and it is an issue with Azure Passes, but nobody is addressing the issues in Microsoft and nobody is testing the changes made to Azure Pass policy and understanding the impact. 

### Lab

Configuring pipelines as code with YAML

### Relevant screenshots

paste here 😉

### Do you want to help us? 👏

- [ ] Create a Pull Request by following this project's [Contribution Guide](https://github.com/MicrosoftLearning/AZ400-DesigningandImplementingMicrosoftDevOpsSolutions/blob/master/.github/CONTRIBUTING.md#how-can-i-collaborate)

**Comment by ArturoBravo111** [comment]
Hi, 

I experienced the same issue this week during my course delivery, exactly as @ElaMark describes. This issue affects multiple labs. 

Problems start appearing in the lab "Configure Pipelines as Code with YAML", specifically in Task 2: Create Azure resources, Step: "To create a Windows App service plan by running the following command".

When students try the following commands:

SERVICEPLANNAME='az400m05l11-sp1'
az appservice plan create --resource-group $RESOURCEGROUPNAME --name $SERVICEPLANNAME --sku B3

They get this error:

![Screenshot 2024-05-28 163840](https://github.com/MicrosoftLearning/AZ400-DesigningandImplementingMicrosoftDevOpsSolutions/assets/34762894/c2101a21-245f-44d7-a466-31332e53139d)

The error says, "The subscription is not allowed to create or update the serverfarm". Initially, only one student received the error, while two could create the service plan without issues. In the next lab, "Control Deployments using Release Gates", the same error appeared for everyone when trying to create an app service plan.

We tried using different regions and different SKUs, but nothing worked. All subsequent labs that use an App Service plan encounter the same error. I ended up creating new subscriptions for my students, but this is not a practical solution.

I observed that when you attempt these labs with a brand new Azure subscription, everything works without issues. It's only after running multiple labs that this error starts appearing.

I agree with @ElaMark, this is not a bug in the labs; it's clearly an issue with Azure Passes that requires an urgent fix. This error didn't occur before.

**Comment by yashints** [comment]
I've raised this internally to see what answers we can get, thanks for your patience

**Comment by LuizMacedo** [comment]
We are still working to find the best way to resolve this issue.

**Comment by ElaMark** [comment]
Hi,

It's been a while now so I thought I might attempt to use an Azure Pass to create an S1 App Service Plan (east us) and a Consumption based Function App (east us).

Both worked for me. If somebody else could confirm this, then the issue can be closed.

Thanks, Mark

**Comment by LuizMacedo** [comment]
Thanks @ElaMark! To give you some status update from our end I'm still working to make our labs work in Cloud Slice so hopefully really soon we won't need Azure Passes and this issue shouldn't happen again. 

@ArturoBravo111 are you still seeing the issue?

**Comment by LuizMacedo** [comment]
I'm closing this issue for now since I haven't seen any instances of it recently. Feel free to open a new issue if it happens again. Thanks!
